Content.Shared/Slippery/SlipperySystem.cs

```diff
@@ -98,7 +116,16 @@ public sealed class SlipAttemptEvent : CancellableEntityEventArgs, IInventoryRel
 }
 
 /// <summary>
-///     This event is raised directed at an entity that CAUSED some other entity to slip (e.g., the banana peel).
+///     Raised on an entity that CAUSED some other entity to slip (e.g., the banana peel).
 /// </summary>
-[ByRefEvent]
-public readonly record struct SlipEvent(EntityUid Slipped);
+public sealed class SlipEvent : CancellableEntityEventArgs
```

Comment:
this should be a separate "slipAttemptEvent" since other system already use this event for different logic.

Reply:
I changed it back and created a new cancellable event inbetween, so no currently implemented stuff is now impacted by this